### DISCUSSION TITLE
docker-compose第一次启动报错，不能使用mysql

### DISCUSSION THREAD

**Discussion by chenzq1604** [hhyo/Archery#1824] [General]
刚下载使用docker-compose启动组件，结果在mysql那一直报错，启动mysql后一直有问题：

`mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/.text': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/__param': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/__mcount_loc': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/.rodata': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/.strtab': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/.symtab': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/.gnu.linkonce.this_module': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/.rodata.str1.8': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/.note.gnu.build-id': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/__kcrctab': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/.exit.text': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/__ksymtab_strings': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/sections/__ksymtab': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/refcnt': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/version': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/uevent': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/holders': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/holders/iTCO_wdt': Read-only file system
mysql        | chown: changing ownership of './sys/module/iTCO_vendor_support/initsize': Read-only file system
mysql        | chown: changing ownership of './sys/hypervisor': Read-only file system
archery      | 2022/09/07 18:19:41 Problem with dial: dial tcp: lookup mysql on 127.0.0.11:53: no such host. Sleeping 1s
archery      | 2022/09/07 18:19:47 Problem with dial: dial tcp: lookup mysql on 127.0.0.11:53: no such host. Sleeping 1s
archery      | 2022/09/07 18:19:53 Problem with dial: dial tcp: lookup mysql on 127.0.0.11:53: no such host. Sleeping 1s
`

不知道是什么原因？大家有碰到吗？

**Comment by chenzq1604** [comment]
使用的包是：
Archery-v1.8.5.tar.gz
还有：
Archery-master.zip

**Comment by chenzq1604** [comment]
`[root@localhost docker-compose]# docker-compose version
Docker Compose version v2.10.2
[root@localhost docker-compose]# docker version
Client: Docker Engine - Community
 Version:           20.10.8
 API version:       1.41
 Go version:        go1.16.6
 Git commit:        3967b7d
 Built:             Fri Jul 30 19:55:49 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.10
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.9
  Git commit:       e2f740d
  Built:            Mon Oct 25 07:43:13 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.9
  GitCommit:        e25210fe30a0a703442421b0f60afac609f950a3
 runc:
  Version:          1.0.1
  GitCommit:        v1.0.1-0-g4144b63
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
`

**Comment by zhu547043537** [comment]
docker-compose第一次启动mysql报错，我遇到的是mysql的5.7的默认版本有问题，docker-compose中的配置文件的Mysql的docker版本加个mysql小版本号就可以了
